examples/llama.vim

```diff
@@ -93,6 +93,33 @@ let s:default_config = {
 
 let g:llama_config = get(g:, 'llama_config', s:default_config)
 
+let s:nvim_ghost_text = exists('*nvim_buf_get_mark')
+let s:vim_ghost_text = has('textprop')
+
+if s:vim_ghost_text
+    let s:hint_hlgroup = 'llama_hl_hint'
+    let s:info_hlgroup = 'llama_hl_info'
+
+    if empty(prop_type_get(s:hint_hlgroup))
+        call prop_type_add(s:hint_hlgroup, {'highlight': s:hint_hlgroup})
+    endif
+    if empty(prop_type_get(s:info_hlgroup))
+        call prop_type_add(s:info_hlgroup, {'highlight': s:info_hlgroup})
+    endif
```

Comment:
These `s:` variables should go inside `llama#init` after `let s:current_job` and before `augroup llama` in order to have them all together in one place.